stackbit.yaml

```diff
@@ -0,0 +1,8 @@
+ssgName: gatsby
+buildCommand: rm -f static/_redirects ; gatsby build
+publishDir: public
+import:
+  type: contentful
+  spaceIdEnvVar: CF_COVID_SPACE_ID
+  accessTokenEnvVar: CF_COVID_ACCESS_TOKEN
```

Comment:
```suggestion
  accessTokenEnvVar: CF_COVID_ACCESS_TOKEN
  accessTokenEnvVar: COVID_COUNTRY
```

Reply:
this to make the stats on the sidebar configurable

Reply:
Hi @Khaledgarbaya,

It seems that `COVID_COUNTRY` variable should be provided by user before creating/building the site. Right now there is no interface in Stackbit to provide additional environment variables and their values when creating new site.

The other two properties `spaceIdEnvVar` and `accessTokenEnvVar` are just a hint to Stackbit on what name to use when it configures Netlify with "space id" and "delivery API token". But the values for these variables Stackbit gets from Contentful API when user connects to Contentful in Stackbit UI.

What I can suggest is moving configuration from environment variable to Contentful to allow user configure it right from Contentful. This could be a little challenging because this value is set in `gatsby-config.js` as an option to `gatsby-source-mathdroid-covid19` before Contentful actually fetched any data. But let me see if I can come up with an easy way to do that. I will submit it as a separate PR.